packages/docs/src/mdx/coreComponents/ExpandableList.mdx

```diff
@@ -96,14 +196,53 @@ export const DemoComponent = ({}) => {
   })
   return (
     <Container>
-      <ExpandableList items={listItems} />
+      <ExpandableList items={listItems} accordion={true} />
+    </Container>
+  )
+}
+
+export const DemoComponentAccordion = ({}) => {
+  const [selectedItem, setSelectedItem] = useState('none')
+  const listItems = ITEMS_ACCORDION.map(item => {
+    return {
+      ...item,
+      selected: selectedItem === item.id,
+      onClick: () => {
+        setSelectedItem(item.id)
+      },
+      items:
+        item.items !== undefined
+          ? item.items.map(subItem => {
+              return {
+                ...subItem,
+                selected: selectedItem === subItem.id,
+                onClick: () => {
+                  setSelectedItem(subItem.id)
+                },
+              }
+            })
+          : undefined,
+    }
+  })
+  return (
+    <Container>
+      <ExpandableList items={listItems} accordion={true} />
     </Container>
   )
 }
 
 export const onClick = () => {}
 
-<DemoComponent />
+<DemoContainer>
+  <DemoContainerItem>
+    <Typography variant="card-title">Expandable list</Typography>
+    <DemoComponent />
```

Comment:
Could you add one more expandable items in this example as well in order to see the differences with another?

Reply:
Reusing same items for both cases now.